support/hololens/ServoApp/Package.appxmanifest

```diff
@@ -27,26 +27,24 @@
         <uap:SplashScreen Image="Assets\SplashScreen.png" />
       </uap:VisualElements>
       <Extensions>
+        <uap5:Extension Category="windows.appExecutionAlias" Executable="ServoApp.exe" EntryPoint="ServoApp.App">
+          <uap5:AppExecutionAlias>
+            <uap5:ExecutionAlias Alias="Servo.exe" />
+          </uap5:AppExecutionAlias>
+        </uap5:Extension>
         <uap:Extension Category="windows.protocol">
           <uap:Protocol Name="fxr">
             <uap:Logo>Assets\StoreLogo.png</uap:Logo>
             <uap:DisplayName>Firefox Reality URL</uap:DisplayName>
           </uap:Protocol>
         </uap:Extension>
-        <uap5:Extension
-          Category="windows.appExecutionAlias"
-          Executable="ServoApp.exe"
-          EntryPoint="ServoApp.App">
-          <uap5:AppExecutionAlias>
-            <uap5:ExecutionAlias Alias="Servo.exe" />
-          </uap5:AppExecutionAlias>
-        </uap5:Extension>
       </Extensions>
     </Application>
   </Applications>
   <Capabilities>
     <Capability Name="internetClient" />
     <Capability Name="codeGeneration" />
     <Capability Name="privateNetworkClientServer" />
+    <Capability Name="internetClientServer" />
   </Capabilities>
-</Package>
+</Package>
```

Comment:
Are these changes just changes to the XML formatting?

Reply:
Visual Studio doing its things.